array-manipulation-2/src/take-right.js

```diff
@@ -0,0 +1,12 @@
+/* exported takeRight */
+function takeRight(array, count) {
+  var takeRightArray = [];
+  for (var i = array.length - count; i < array.length; i++) {
+    if (array.length > 0) {
+      takeRightArray.push(array[i]);
+    } else {
+      break;
+    }
+  }
```

Comment:
The `if/else` in here is not necessary. Try to avoid even starting the loop at all if the array is empty.